features/step_definitions/branch_steps.rb

```diff
@@ -17,7 +17,7 @@
   create_branch branch_name, remote: !local
   if relation
     commit_to_branch = relation == 'behind' ? 'main' : branch_name
-    create_commits branch: commit_to_branch
+    create_commits branch: commit_to_branch, message: "#{relation} #{branch_name} commit"
```

Comment:
I don't like the commit messages added by this. I think we should just get rid of these steps and set up the commits manually. since the commit messages are now coming into play